sdk/identity/Azure.Identity.BrokeredAuthentication/api/Azure.Identity.BrokeredAuthentication.net462.cs

```diff
@@ -3,10 +3,12 @@ namespace Azure.Identity.BrokeredAuthentication
     public partial class InteractiveBrowserCredentialBrokerOptions : Azure.Identity.InteractiveBrowserCredentialOptions
     {
         public InteractiveBrowserCredentialBrokerOptions(System.IntPtr parentWindowHandle) { }
+        public bool? IsMsaPassthroughEnabled { get { throw null; } set { } }
```

Comment:
This file should not appear in the PR.   Please rebase.

Reply:
This file gets updated due to eng\scripts\Update-Snippets.ps1. If I don't run this, my merge would fail. Please let me know the proper steps to move forward.

Reply:
Your repository is out of date.  You'll need to fetch the latest from main and rebase your changes.

Reply:
reverted